README.md

```diff
@@ -1,7 +1,8 @@
 # drift-backend
-host drift analysis service
+Host drift analysis service
```

Comment:
System unless you want to fix this later?

Reply:
ya, will fix later as part of other card